locust/runners.py

```diff
@@ -59,6 +59,7 @@
 CPU_MONITOR_INTERVAL = 5.0
 HEARTBEAT_INTERVAL = 1
 HEARTBEAT_LIVENESS = 3
+HEARTBEAT_DEAD = -10
```

Comment:
Feels weird that this is a negative number?

Reply:
It symbolic to time with no response, as current heartbeat logic subtracts 1 for each missed heartbeat. Didn't want to introduce another variable to maintain skipped heart beat.